app/code/BVMyBlog/Blog/Api/BlockRepositoryInterface.php

```diff
@@ -0,0 +1,62 @@
+<?php
+declare(strict_types=1);
+
+namespace BVMyBlog\Blog\Api;
+
+use BVMyBlog\Blog\Api\Data\BlockInterface;
+use Magento\Framework\Api\SearchCriteriaInterface;
+use Magento\Framework\Api\SearchResultsInterface;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NoSuchEntityException;
+
+/**
+ * Repository interface post data.
+ */
+interface BlockRepositoryInterface
```

Comment:
Please ensure that interface thrown exceptions and implementation are correspond each other